### DISCUSSION TITLE
SaveBinary doesn't work?

### DISCUSSION THREAD

**Discussion by ztdepztdep** [CFD-GO/TCLB#373] [General]
I use the following to save the U and Rho component. 
          <SaveBinary Iterations="10000" file="bindary" filename="/home/ztdep/2T/TCLB" comp="U,Rho"/>

 But it feeds back that 
Saving component U,Rho to file output/drivencavity-2D_bindary_U,Rho_0.comp
  [  ]    ---- : ...not saved U,Rho

**Comment by llaniewski** [comment]
SaveBinary is for saving internal data to disk.

If you want to export results, like Rho and U, use VTK (or hdf5) output.

**Reply by ztdepztdep** [reply]
I want to restart my computation. so i want to save u and rho to disk , and then load it again.

**Reply by TravisMitchell** [reply]
You can use the checkpoint feature to do this https://github.com/CFD-GO/TCLB/pull/359 

There is an example: example/flow/2d/karman_checkpointExample.xml   this will output an xml file along with your VTK files that you can launch to continue the simulation from a designated checkpoint.
